test/e2e/storage/volume_provisioning.go

```diff
@@ -251,6 +261,101 @@ func checkGCEPD(volume *v1.PersistentVolume, volumeType string) error {
 	return nil
 }
 
+func testZonalDelayedBinding(c clientset.Interface, ns string, pvcCount int) {
+	tests := []testsuites.StorageClassTest{
```

Comment:
`testZonalDelayedBinding()` and `testZonalDelayedBindingWithAllowedTopology()` share a lot of code, and their `StorageClassTest`s are the same too. I was thinking we could combine these two functions